### PR TITLE
Updated and simplified onboarding pages

### PR DESCRIPTION
**Pull request by JunMagic88**

### CODE REVIEW

**Comment by JunMagic88** [comment]
Updated and simplified Built on the Blockchain and Seeding the Cosmos

**Review by JunMagic88 — COMMENTED** [review]
Updated onboarding pages

File: docs/_onboard/seeding-the-cosmos.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ last_modified_date: 2022-12-01
 Seeding the Cosmos
 =======================
 
-Y-Foundry DAO is a community born out of frustration with the often extractive and insecure nature of crypto. Manipulation infects systems advertised as trustless, leading to unintended consequences and losses. Cults of personalities develop and take advantage of the uninformed.
+Y-Foundry DAO is a community born out of frustration and contempt with opague and exploitative crypto projects. These projects are often run by cult-like personalities who extracts value from uninformed participants to enrich their own pockets, while avoiding transparency and accountability for their actions and decisions.
```
Comment (crashtestdumb1) [inline docs/_onboard/seeding-the-cosmos.md:12]: "opague" should be "opaque"
Not sure if "cult-like personalities" is accurate. Cult-like would be describing the followers not the leaders. Maybe "personalities who abuse a cult-like following with their uninformed participants to enrich their own pockets..."

**Review by crashtestdumb1 — CHANGES_REQUESTED** [review]
Just a couple of comments. Overall very nice. I appreciate the more concise language and removing some of the content that is discussed in later sections.

File: docs/_onboard/seeding-the-cosmos.md (hunk 1 above)
Comment (JunMagic88) [reply docs/_onboard/seeding-the-cosmos.md]: Discussed with Ray - this is an old page and is not longer in /docs. should be removed

File: _drafts/about/about.md [hunk 2]
```diff
@@ -12,20 +12,18 @@ last_modified_date: 2022-10-25
...
+Creators can work with their community to refine their project ideas before submitting a proposal on the YFD platform. The community can then review the proposal, provide the necessary funding to help the project launch, and become a Supporter sharing the project's success. Whitelisted builders from within the YFD community can accept bounties to help build the project.
 
 ## The unique value of Y-Foundry DAO.
 
-_The ability for the protocol to self-fund development without dilution of its token/governance power._
+_Customisable on-chain fund-raising and governance mechanisms that enables transparency and accountability throughout the project development lifecycle. See [How Y-Foundry DAO Governance Works](../../govern/participate/)_
 
-_The decoupling of Governance and the Treasury, such that the Treasury is a smart contract accountable to perform actions as determined by on-chain governance._
+_Trustless execution of treasury actions based on on-onchain votes - no one has privileged access, not even the core team._
```
[the remark this answers is not in the document]
Comment (JunMagic88) [reply _drafts/about/about.md:27]: Discussed with Ray - this is an old page and should be removed.

**Review by crashtestdumb1 — APPROVED** [review]

**Pull request merged**